### DISCUSSION TITLE
What is a place to hang-out & discuss the project

### DISCUSSION THREAD

**Discussion by Anton-Latukha** [haskell/haskell-language-server#2209] [Q&A]
I want to make some fixes to HLS and use that as an introduction into from time to time doing work on HLS.

As a maintainer I know how is it to do quality work on a pretty complex project. And how much time goes into learning ins and outs of it.
I can spend evenings (really much more) learning the project, before I would know proper places to fix abstract representations.

Or I can just ask maintainers/people for directions - to point to module/s should be enough for me.

But this is not about me or my things, used it just as an example prelude.

On the project, main page, In `"Contributing"` or `"Troubleshooting"` or in `ReadTheDocs TOC` seems to be no indication of any chat room where people can hang-out, and ask such questions as mine to help one another.

Where to go to get some basic help while wanting to work on the project?

**Comment by jneira** [comment]
Hi, happy to see new contributors, we have some links about here: https://haskell-language-server.readthedocs.io/en/latest/contributing/contributing.html#contributing-guidelines
Maybe they shouls be more visible in indices?

> Join our [IRC channel](https://web.libera.chat/?channels=#haskell-language-server) at #haskell-language-server on libera.
> Follow the [Haskell IDE team twitter account](https://twitter.com/IdeHaskell) for updates and help.
> Join the [#haskell-tooling channel](https://discord.com/channels/280033776820813825/505370075402862594/808027763868827659) in the Functional Programming discord server. You can join the server via [this invitation](https://discord.gg/9spEdTNGrD).

You also can be in touch with some hls maintainers in matrix: https://matrix.to/#/#haskell-tooling:matrix.org
The platform where most maintainers are present at this moment is irc

**Reply by Anton-Latukha** [reply]
Well, as I said. Just was puzzled not seeing the contact information.
I consider it is in the relatively proper place, but it is currently at least 1 step too far into the structure.

People are currently generally accustomed to search contact information on the project frontpage.
For example in [HNix](https://github.com/haskell-nix/hnix) it is in the badge (for badge lovers, because I got users looking for contact info in badges), and in the `#Contributing` section. People (meaning I) also look at the top and the bottom of the frontpage. And then in GitHub frontpage situation people frequently just `ctrl+f` search for it over the information. But use search engine to search over the whole ReadTheDocs book to find contacts - is quite too much.

The think with HLS, the current contributor/user flow is:
1. Search on the front page. Get nothing.
2. Guess right that it may be in `Contribution`, but going there returns again - nothing.
3. Be persistent enough to think about starting reading the whole thing and maybe somewhere there would be information, so opening "Contributing guidelines" and suddenly to the name of the article - find contact info there.

Maybe put the contact info inside the "Contribution" section, or create subsection "Getting help/contact" there, if you want to be on a very safe side - add an ask to understand the "Contribution guidelines" before asking help or questions in the chat.

**Reply by Anton-Latukha** [reply]
Also want to note a good maintainer experience from you, thank you.

**Reply by jneira** [reply]
thanks for suggestions, add a badge is a good idea and I think the readme could have a second level index, where the link to contact section could be shown
